classes/submission/reviewAssignment/Collector.php

```diff
@@ -435,8 +445,24 @@ public function getQueryBuilder(): Builder
 
         $q->when(
             $this->reviewerIds !== null,
-            fn (Builder $q) =>
-            $q->whereIn('ra.reviewer_id', $this->reviewerIds)
+            fn (Builder $q) => $q
```

Comment:
Only open question in my mind is what to do about scenarios when theoretically (no idea if that could happen practically) same reviewer could by assigned both in internal/external review stage in OMP.  And in such scenario we might want to separate and add couple stageId conditions. But will ask first Devika what would be her expectation how that should work and whether it should be considered at all.

Reply:
According to the current logic, just applying the stage ID filter for OMP should be enough, will check. 
For action required by me, in my mind is additionally check submission's relevant stage.

Reply:
@Vitaliy-1 But not sure when/if to apply such filter.. because we are showing review assignments regardless of what stage the submission are in..  And I don't think we have specific views for OMP..

Reply:
Devika is suggesting just to keep it together. So if reviewer teoretically would have review assignment both in internal and external review stage - we would just show the latest thing.

So I can imagine either just find max round_id, I think we can assume these will always be higher with new rounds. Or somehow combine stage and round.